src/registrar/tests/test_views.py

```diff
@@ -485,6 +490,7 @@ def test_application_form_started_allsteps(self):
         # type_page = home_page.click("Edit")
 
         session_id = self.app.cookies[settings.SESSION_COOKIE_NAME]
+        session_wizard = self.app.session["wizard_application"]
```

Comment:
This guy seems to be causing linter to fail, since it's been added and is not used.

Reply:
Yep already fixed that, was using the sesison wizard before to fix the testing issue then found it wasn't needed